### ISSUE TITLE
Remove feature/iot_pnp branch

### ISSUE THREAD

**Issue by jspaith** [Azure/azure-sdk-for-c#1900]
Now that `feature/iot_pnp` branch has been merged into `main`, we should remove `feature/iot_pnp` branch.

We'll need to make sure customers we know of are successfully migrated before we do this.  We'll may need help from Azure SDK repo admin since the branch is locked.

**Comment by ahsonkhan** [comment]
Before deleting it, we should consider updating the release notes for the 1.1.0 release and update the link to the release tag (or remove it all together):
https://github.com/Azure/azure-sdk-for-c/releases/tag/1.1.0

cc @danieljurek, @weshaggard

**Comment by ahsonkhan** [comment]
> Before deleting it, we should consider updating the release notes for the 1.1.0 release and update the link to the release tag (or remove it all together): https://github.com/Azure/azure-sdk-for-c/releases/tag/1.1.0
> 
> cc @danieljurek, @weshaggard


@danieljurek, we have removed that feature branch, so the link in the release notes now 404s. It isn't crucial, but is there any way to retroactively fix-up the release notes for the 1.1.0 release to remove the link (on GitHub at least, if not the published doxygen or release blogs)? We have shipped this feature in 1.2.0 now.

Maybe something like this:

```diff
- These will ship in a future release and will continue to be available as beta from this feature branch.
+ These will ship in a future release.
```

**Comment by weshaggard** [comment]
@ahsonkhan are you talking about the release notes at https://github.com/Azure/azure-sdk/pull/3723?
